src/content/guides/tenant-cluster-labelling/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Labelling tenant clusters"
+description: "Introduction to labelling tenant clusters"
+date: "2020-04-27"
+type: page
+weight: 130
+tags: ["recipe"]
+---
+
+# Labelling tenant clusters
+
+It is possible to assign *key value labels* to Giant Swarm tenant clusters on AWS with release version 10.0.0 and above.
+
+Labels are a mechanism to assign short pieces of additional information to your Giant Swarm tenant clusters.
+Under the hood, tenant cluster labels are Kubernetes labels attached to [`Cluster`] (`clusters.cluster.x-k8s.io`) resources.
+Therefore, all means of listing tenant cluster labels will return all Kubernetes labels attached to [`Cluster`] resources requested.
+Label keys and values are freely modifiable except labels with keys containing `giantswarm.io`.
+
+Working with tenant cluster labels works likewise as working with Kubernetes labels.
+More information about Kubernetes Labels can be found in the [Kubernetes Labels and Selectors] documentation.
+
+## Working with tenant cluster labels using the Giant Swarm API
+
+Tenant cluster labels of clusters with release version 10.0.0 and above are returned by executing a [getClusters] request.
+The field `labels` of suitable tenant clusters contains the labels currently attached to the cluster.
```

Comment:
I can't see a mention of the endpoint https://docs.giantswarm.io/api/#operation/getClusterLabels. I think it's worthwhile mentioning here.